### PR TITLE
Simplified Derivation of the Cross-Entropy Loss Gradient (for a kid)

### LINKED ISSUE #10: Simplified Derivation of the Cross-Entropy Loss Gradient (for a kid)
In this PR, i have tried to explain cross entropy loss and it's gradient calculation in oder to make it even understandable to a kid.

### PR DESCRIPTION
**Pull request by sakshamlakhera**
Fixes #10

### CODE REVIEW

File: books/pattern-classification/bonus/cross_entropy_gradient.html [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+            tex: {
+                inlineMath: [
+                    ["$", "$"],
+                    ["\\(", "\\)"],
+                ],
+            },
+        };
+    </script>
+    <script defer src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+    <style>
```
Comment (kkodweis2013) [inline books/pattern-classification/bonus/cross_entropy_gradient.html:20]: No inline styling. Please refer to the existing .css style assets. If you want to implement something new/edit, please do so and we will review upon submission.

File: index.html [hunk 2]
```diff
@@ -63,7 +63,12 @@ <h2>📚 Books</h2>
                   <a
                     href="books/pattern-classification/chapters/02-bayes-decision.html"
                   >
-                    Chapter&nbsp;2: Bayes Decision Theory
+                    Chapter&nbsp;2: Bayes Decision Theory
+                  </a>
+                </li>
+                <li>
+                  <a href="books/pattern-classification/bonus/cross_entropy_gradient.html">
+                    Bonus Read: Simplified Derivation of the Cross-Entropy Loss Gradient
```
Comment (kkodweis2013) [inline index.html:71]: Please remove "Bonus Read:".

Comment (kkodweis2013) [reply index.html:71]: Please add to the newly created Mathmatical Foundations section

**Review by kkodweis2013 — CHANGES_REQUESTED** [review]
Please update and send back through, this looks great and I can't wait to include it in our project and get your name on the contributors list.

File: index.html (hunk 2 above)
Comment (sakshamlakhera) [reply index.html]: Changes Done.

**Comment by COD1995** [comment]
Please see #38 for the correct workflow so the admin has a clearer understanding of what is happening.

**Review by kkodweis2013 — APPROVED** [review]
Changes look good. Approved
